debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/util/TestHelper.java

```diff
@@ -112,6 +112,15 @@ public static JdbcConfiguration defaultJdbcConfig() {
                 .build();
     }
 
+    public static JdbcConfiguration jdbcConfig(String hostname, int port, String user, String password) {
```

Comment:
Could you please call `defaultConnectorConfig` and just override user and password and keep host/port untouched?